server/README.md

```diff
@@ -315,7 +315,17 @@ containing a `crt` file and point `X509_CA_BUNDLE` environmental variable to tha
 
 NOTE: See `openshift-examples` directory for an out of the box setup for OpenShift.
 
+### Enable some metrics
 
+Keycloak image can collect some statistics for various subsystem which will then be available in the management console and the `/metrics` endpoint.
+You can enable it with the following environment variables :
+* `KEYCLOAK_DATASOURCES_STATISTICS_ENABLED` for the `datasources` subsystem
+* `KEYCLOAK_UNDERTOW_STATISTICS_ENABLED` for the `undertow` subsystem
+* `KEYCLOAK_JGROUPS_STATISTICS_ENABLED` for the `jgroups` subsystem
+
+You can also enable all of those with `KEYCLOAK_STATISTICS_ENABLED`.
```

Comment:
This is currently WILDFLY_STATISTICS_ENABLED in xml file right? So would need to change the value in statistics.cli

Reply:
No all was changed with KEYCLOAK prefix following your first comment.

Reply:
Never mind. I didn't look properly. I thought there was already a WILDFLY_STATISTICS_ENABLED env variable, but I see you've added it.